routes/auth.js

```diff
@@ -16,19 +16,21 @@ module.exports = (app, passport) => {
  *     summary: Register a new user account
  *     description: Creates a new user account using the provided user data.
  *     tags:
- *       - Registration
+ *       - Auth
  *     parameters:
  *       - in: body
- *         name: user
- *         description: User object containing registration details
+ *         name: credentials
```

Comment:
prima je krivi objekt.. kad se registriras, ne saljes id od usera koji se kreira u bazi.. korisnik ne zna za to.. korisnik salje username i password i aplikacija sama u pozadini kreira usera... pa sam i to ispravija